src/main/webapp/app/components/data-table/data-table.component.ts

```diff
@@ -220,11 +247,22 @@ export class DataTableComponent implements OnInit, OnChanges {
 
     /**
      * Performs a case-insensitive search of "word" inside of "text".
+     * If "word" consists of multiple segments separated by a space, each one of them must appear in "text".
+     * This relaxation has the benefit that searching for "Max Mustermann" will still find "Max Gregor Mustermann".
+     * Additionally, the wildcard symbol "*" is supported.
      *
      * @param text string that is searched for param "word"
      */
     private foundIn = (text: string) => (word: string) => {
-        return word && text.toLowerCase().includes(word.toLowerCase());
+        const segments = word.toLowerCase().split(' ');
+        return (
+            text &&
+            word &&
+            segments.every(segment => {
```

Comment:
I added support for a single character match via `?`.

I was aware that the highlighting no longer works when using wildcard symbols. Unfortunately this is not an easy fix since the highlighting is done internally by a [third-party component](https://ng-bootstrap.github.io/#/components/typeahead/api).

In this use case, I don't think the highlighting is particularly important (since the text to be searched is usually very short) and losing it in the rare case that someone actually performs an advanced search with wildcard symbols seems tolerable to me.

Reply:
Thank you for implementing the support for `?`. I works as expected on the Test Server. 

Yes the issue with highlighting is not critical and it doesn't make sense to put to much time into fixing it.

Reply:
And please also update the PR description. Mention that you added `?` character

Reply:
Will do.